flutter/example/integration_test/integration_test.dart

```diff
@@ -0,0 +1,109 @@
+import 'dart:io';
+
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+import 'package:sentry_flutter_example/main.dart';
+
+void main() {
+  final binding = IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+  binding.framePolicy = LiveTestWidgetsFlutterBindingFramePolicy.fullyLive;
+
+  Future<void> setupSentryAndApp(WidgetTester tester) async {
+    await setupSentry(() async {
+      await tester.pumpWidget(DefaultAssetBundle(
+        bundle: SentryAssetBundle(enableStructuredDataTracing: true),
+        child: MyApp(),
+      ));
+      await tester.pumpAndSettle();
+    });
+  }
+
+  // Tests
+
+  testWidgets('setup sentry and render app', (tester) async {
+    await setupSentryAndApp(tester);
+
+    // Find any UI element and verify it is present.
+    expect(find.text('Open another Scaffold'), findsOneWidget);
+  });
+
+  testWidgets('setup sentry and capture event', (tester) async {
+    await setupSentryAndApp(tester);
+
+    final event = SentryEvent();
+    final sentryId = await Sentry.captureEvent(event);
+
+    expect(sentryId != SentryId.empty(), true);
+  });
+
+  testWidgets('setup sentry and capture exception', (tester) async {
+    await setupSentryAndApp(tester);
+
+    final exception = SentryException(
+        type: 'StarError',
+        value: 'I have a bad feeling about this...'
+    );
+    final sentryId = await Sentry.captureException(exception);
```

Comment:
Lets throw and do a try/catch here because we can pass the stackTrace optional param as well